.github/workflows/docbuild.yml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: curl https://raw.githubusercontent.com/osmosis-labs/osmosis/main/x/pool-incentives/spec/README.md > developing/osmosis-core/modules/spec-pool-incentives.md
 
       - name: Download mint docs
-        run: curl https://raw.githubusercontent.com/osmosis-labs/osmosis/main/x/mint/spec/README.md > developing/osmosis-core/modules/spec-mint.md
```

Comment:
We should make this change on the stage doc build as well in order to preview changes.